src/main/webapp/resources/js/pages/admin/components/user/CreateNewUser.jsx

```diff
@@ -0,0 +1,301 @@
+  useGetSystemRolesQuery,
+} from "../../../../apis/settings/settings";
+import { SPACE_SM } from "../../../../styles/spacing";
+import { PagedTableContext } from "../../../../components/ant.design/PagedTable";
+import { AddNewButton } from "../../../../components/Buttons/AddNewButton";
+import { ScrollableModal } from "../../../../components/ant.design/ScrollableModal";
+import { validatePassword } from "../../../../utilities/validation-utilities";
+import { PasswordPolicyAlert } from "../../../../components/alerts/PasswordPolicyAlert";
+
+/**
+ * React component to display the create new user form.
+ * @returns {*}
+ * @constructor
+ */
+export default function CreateNewUser() {
+  const { updateTable } = React.useContext(PagedTableContext);
+  const { data: locales = [] } = useGetLocalesQuery();
+  const { data: systemRoles = [] } = useGetSystemRolesQuery();
+  const { data: emailConfigured = false } = useGetEmailConfiguredQuery();
+  const [createUser] = useCreateNewUserMutation();
+  const [form] = Form.useForm();
+  const [activationEmail, setActivationEmail] = React.useState(emailConfigured);
+  const [visible, setVisibility] = React.useState(false);
+  const [submitting, setSubmitting] = useState(false);
+  const [mailFailure, setMailFailure] = useState(false);
+  const usernameInput = React.useRef();
+  const isAdmin = window.TL._USER.systemRole === "ROLE_ADMIN";
+
+  useEffect(() => {
+    if (visible) {
+      usernameInput.current.focus();
+    }
+  }, [visible]);
+
+  const onFormFinish = (values) => {
+    setSubmitting(true);
+    createUser({ ...values })
+      .unwrap()
+      .then(() => {
+        notification.success({
+          message: i18n("CreateNewUser.notification.success"),
+          className: "t-user-page-notification-success",
+        });
+        form.resetFields();
+        setVisibility(false);
+        setMailFailure(false);
+        updateTable();
+      })
+      .catch((error) => {
+        notification.error({
+          message: i18n("CreateNewUser.notification.error"),
+        });
+        if (error.status === 400) {
+          const fields = Object.entries(error.data.errors).map(
+            ([field, error]) => ({
+              name: field,
+              errors: [error],
+            })
+          );
+          form.setFields(fields);
+        } else if (error.status === 409) {
+          form.setFieldsValue({ activate: false });
+          setActivationEmail(false);
+          setMailFailure(true);
+        }
+      })
+      .finally(() => setSubmitting(false));
+  };
+
+  return (
+    <>
+      <AddNewButton
+        className="t-add-user-btn"
+        onClick={() => setVisibility(true)}
+        text={i18n("CreateNewUser.button")}
+      />
+      <ScrollableModal
+        title={i18n("CreateNewUser.title")}
+        onCancel={() => setVisibility(false)}
+        visible={visible}
+        maxHeight={window.innerHeight - 250}
+        width={640}
+        footer={
+          <Button
```

Comment:
As I said in the other comment, please use the okBtnProps to handle the buttons, there is no need to make your own here, and also you should have the cancel button.

Reply:
Please see 07095c6b09dabf2f7691451fcca24306043fc044.